google-cloud-firestore/src/main/java/com/google/cloud/firestore/BatchWriteResult.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.firestore;
+
+import com.google.cloud.Timestamp;
+import javax.annotation.Nullable;
+
+/**
+ * A BatchWriteResult wraps the write time and status returned by Firestore when making
+ * BatchWriteRequests.
+ */
+public final class BatchWriteResult {
```

Comment:
This class appears to be an internal class so we can scope down the visibility and annotate as such.

```suggestion
@IntenralApi
final class BatchWriteResult {
```

Reply:
done.